README.md

```diff
@@ -1,71 +1,50 @@
-# Nuked SC-55 CLAP audio plugin
+# Nuked SC-55 CLAP
 
-The plugin is built upon [J.C. Moyer's fork](https://github.com/jcmoyer/Nuked-SC55)
-of nukeykt's original [Nuked SC-55](https://github.com/nukeykt/Nuked-SC55)
-project.
-
-The plugin aims to preserve an important part of DOS gaming history for all to
-freely enjoy for posterity. It's only intended for **personal use** (e.g.,
-retro gaming or writing music as a hobby) and **research purposes**.
-
-As per the original Nuked SC-55 license, neither the code nor the published
-binaries may be used directly or indirectly for the creation of commercial
-Roland SC-55 emulation hardware boxes. Moreover, any use of the software in
-commercial music production is prohibited, and so is including the plugin in
-any commercial software package.
+Nuked SC-55 for the CLever Audio Plug-in ([CLAP](https://cleveraudio.org/)) is built upon [J.C. Moyer's fork](https://github.com/jcmoyer/Nuked-SC55) of nukeykt's original [Nuked SC-55](https://github.com/nukeykt/Nuked-SC55) project.
 
+The plug-in aims to preserve an important part of DOS gaming history for all to freely enjoy for posterity. It is only intended for **personal use** (e.g., retro gaming or writing music as a hobby) and **research purposes**. See the [License](#license) section for additional details.
 
 ## Installation
 
-Download the latest version for your operating system from the [releases
-page](https://github.com/johnnovak/Nuked-SC55-CLAP/releases/) page, then unzip
-it into one of these OS-specific locations:
+Download the latest version for your operating system from the [releases page](https://github.com/johnnovak/Nuked-SC55-CLAP/releases) page, then unzip it into one of these OS-specific locations:
 
 - **Windows**
+
   - `C:\Program Files\Common Files\CLAP\`
-  - `$LOCALAPPDATA/Programs/CLAP`
+  - `$LOCALAPPDATA\Programs\CLAP\`
 
 - **macOS**
+
   - `/Library/Audio/Plug-Ins/CLAP/`
   - `$HOME/Library/Audio/Plug-Ins/CLAP/`
 
 - **Linux**
+
   - `/usr/lib/clap/`
   - `$HOME/.clap/`
 
-
 ### macOS Gatekeeper
 
+If macOS Gatekeeper prevents the plug-in from running, you will need to explicitly whitelist it with the following command from the terminal (replace `<path-to>` with the actual path to `Nuked-SC55.clap`):
 
-If macOS Gatekeeper doesn't let you open the plugin, you'll need to explicitly
-whitelist it with the following command from the terminal (replace `<path-to>`
-with the actual path to `Nuked-SC55.clap`):
-
-
-```bash
+```zsh
 sudo xattr -rd com.apple.quarantine <path-to>/Nuked-SC55.clap
 ```
 
+### ROM Files
```

Comment:
Only the first letter should be uppercase (no title case capitalisation, please).